README.md

```diff
@@ -92,12 +92,25 @@ attestations (and related assertions; see below).
 
 This begs the question: How to enable key attestation on iOS?
 After all, many applications exist, which require some proof that a key used for critical operations resides in hardware.
-<br>
-Here, the ability to obtain a so-called *assertion* comes to the rescue: iOS allows generating an *assertion* for some
+
+#### Legacy Attestation Format (Deprecated, but still Supported since Version 2.2.0)
+To emulate key attestation, the ability to obtain a so-called *assertion* comes to the rescue: iOS allows generating an *assertion* for some
 data by signing it using the same key backing a previously obtained attestation.
 By that logic, computing an assertion over the public key of a freshly generated public/private key pair proves that an
 authentic, uncompromised app on a non-jailbroken device was used to generate this key pair as intended by the app developer.
 
+#### Supreme Attestation Format (Supported Since Version 2.2.0)
+Following Apple's attestation format makes is clear, that no data, but only hashes are ever encoded and signed.
+Hence, it allows for a lot of flexibility when it comes to the data to be hashed.
+The new supreme attestation format exploits this and does not only pass the hash over a challenge to the AppAttest
+service, but instead constructs a structured (JSON) client data object, inspired by WebAuthn and passes tha hash of this
+data to `DCAppAttest`.
+As such, clients need to create a Secure-Enclave-protected key pair before callinf `DCAppattest` and construct the structured
+client data, containing the public part of this key pair and the server challenge.
+This structured data format is defined in the _Signum's
+[Indispensable](https://a-sit-plus.github.io/signum/dokka/indispensable/at.asitplus.signum.indispensable/-ios-homebrew-attestation/index.html)_
```

Comment:
but the webpage behind that link does not show an example

Reply:
fixed and rephrased slightly